_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -564,11 +568,17 @@ public static function register_endpoints() {
 	 *
 	 * @return bool true.
 	 */
-	public static function update_setup_questionnaire( $request ) {
-		// TODO: add validation.
```

Comment:
I think we'll still need to add some validation here for `$questionnaire` before we save it?

Reply:
I've added validation now.